### ISSUE TITLE
Test failure due to missing `permission_test` file

### ISSUE THREAD

**Issue by orhun** [kyoheiu/felix#200]
Hey, Arch maintainer of `felix` here! 👋🏼

I'm getting the following test failure while building `2.2.7`:

```
running 13 tests
test functions::tests::test_display_count ... ok
test functions::tests::test_convert_to_permissions ... ok
test functions::tests::test_duration_to_string ... ok
test functions::tests::test_format_time ... ok
test functions::tests::test_proper_size ... ok
test functions::tests::test_list_up_contents ... FAILED
test functions::tests::test_split_str_including_wide_char ... ok
test magic_image::tests::test_inspect_image ... ok
test state::tests::test_has_write_permission ... FAILED
test run::tests::zoxide_test ... ok
test magic_packed::tests::test_inspect_signature ... ok
test state::tests::bench_update_single ... ok
test state::tests::bench_update_parallel ... ok

failures:

---- functions::tests::test_list_up_contents stdout ----
thread 'functions::tests::test_list_up_contents' panicked at 'assertion failed: `(left == right)`
  left: `"├ archives\n└ images"`,
 right: `"├ archives\n├ images\n└ permission_test"`', src/functions.rs:338:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- state::tests::test_has_write_permission stdout ----
thread 'state::tests::test_has_write_permission' panicked at 'called `Result::unwrap()` on an `Err` value: Io("No such file or directory (os error 2)")', src/state.rs:1754:52


failures:
    functions::tests::test_list_up_contents
    state::tests::test_has_write_permission

test result: FAILED. 11 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.04s

error: test failed, to rerun pass `--bin fx`
```

Currently I'm doing the following as a workaround:

```
touch testfiles/permission_test
chmod 444 testfiles/permission_test
```

<s>Can you maybe add this file to the repository?</s> Related pull request: #201

**Comment by kyoheiu** [comment]
Ah, I didn't notice that empty directory is not detectable on remote... Adding empty text file, and test with `exacl` seem doing right. #202

**Comment by kyoheiu** [comment]
#202 merged, now I think you can pass the test. Could you please check?

**Comment by orhun** [comment]
This time I'm getting this:

```
running 13 tests
test functions::tests::test_duration_to_string ... ok
test functions::tests::test_convert_to_permissions ... ok
test functions::tests::test_proper_size ... ok
test functions::tests::test_format_time ... ok
test functions::tests::test_display_count ... ok
test functions::tests::test_split_str_including_wide_char ... ok
test functions::tests::test_list_up_contents ... ok
test magic_image::tests::test_inspect_image ... ok
test state::tests::test_has_write_permission ... FAILED
test run::tests::zoxide_test ... FAILED
test magic_packed::tests::test_inspect_signature ... ok
test state::tests::bench_update_single ... ok
test state::tests::bench_update_parallel ... ok

failures:

---- state::tests::test_has_write_permission stdout ----
thread 'state::tests::test_has_write_permission' panicked at 'assertion failed: !has_write_permission(p.as_path()).unwrap()', src/state.rs:1762:9

---- run::tests::zoxide_test stdout ----
thread 'run::tests::zoxide_test' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/run.rs:1703:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    run::tests::zoxide_test
    state::tests::test_has_write_permission

test result: FAILED. 11 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.04s

error: test failed, to rerun pass `--bin fx`
```

**Comment by kyoheiu** [comment]
Thank you for testing.
Fix #203 merged:

- Remove `zoxide_test`, This checked if zoxide query works, which has nothing with felix itself.
- In `test_has_write_permission` use `chmod 444` instead of `exacl::setfacl`

How about this?
(Note that if you run the test as root, `test_has_write_permission` will fail)

**Comment by orhun** [comment]
Yeah, that works. 🥳 Thanks, closing...
